config/manager/patches/add_root_certificates.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ocm-controller
+  labels:
+    control-plane: controller
+spec:
+  selector:
+    matchLabels:
+      app: ocm-controller
+  template:
+    spec:
+      containers:
+        - name: manager
+          image: open-component-model/ocm-controller
+          volumeMounts:
+            - mountPath: "/etc/ssl/certs"
+              name: "root-certificate"
+            - mountPath: "/certs"
+              name: "client-certificates"
+      volumes:
+        - name: "root-certificate"
+          secret:
+            secretName: "developer-root-certificate"
+            items:
+              - key: "ca-certificates.crt"
+                path: "ca-certificates.crt"
+        - name: "client-certificates"
+          secret:
+            secretName: "registry-certs"
+            items:
+              - key: "server.pem"
```

Comment:
for e2e it's fine to provide same `pem` for server and client, but for end users I think those should be different.

Reply:
That is none of our concern though, right?

Reply:
I guess not here, but on cli side yeah.